### ISSUE TITLE
Target Audience

### ISSUE THREAD

**Issue by Jutanium** [solidjs/solid-docs-legacy#34]
I'd love to start jumping in and improving the documentation, especially the tutorial, which is an incredible accomplishment so far but can benefit from some changes (mainly streamlining to make it less wordy and to assume less knowledge on the reader).

Along those lines, I can't seem to find any documentation here on the intent of each part of the docs. The main parts seem to be Getting Started, Tutorial, and the full Documentation. It's the former two that I think it would be really helpful to have clarity on as we move forward. 

What is the relationship between the Getting Started guide and the Tutorial? It's clear that the Tutorial is more comprehensive, and the Getting Started guide is more of an overview. However, it would help to write out the goals of each. For what it's worth, my initial take is that the Getting Started guide should be the initial stop for someone who knows other reactive frameworks and wants to learn Solid, while the Tutorial should be an in-depth guide for a complete beginner who wants to go from zero to hero with Solid. 

Along those lines, I think the tutorial should be as accessible as possible. There's no egregious problem with it or anything, but it does sometime assume knowledge of terminology or predecessor concepts. (A good example is the "What is Solid" description on the first introductory lesson). I'd love to help improve that; determining the goals for the Tutorial will help contributors to be aligned as we work on it.

**Comment by davedbase** [comment]
Yes! I support this. The tutorials and docs could totally benefit from someone to help tailor the wording and explanation to an audience. To help explain our logic to date:

* Guides are summaries of high level concepts. They give you the broad strokes
* Docs are purely focused on explaining mechanics and act as a reference. They should be short and concise enough to give you quick answers
* Tutorials are purely about being learning material

As mentioned on Discord the concern I have with refactoring docs is that we have a considerable amount of support with translations. In regards to changes this is the process I'm hoping to see:

1. Language specific rewording should be more about word-smithing, the context and thrust of the content section by section should remain mostly the same
2. Additions of sections need to be copied into the other languages and marked for translation so that a native speaker can support us on the adjustment (we have a lot of fantastic people who jump in to support on that.
3. We can consider removing/completely refactoring sections just the same way we manage additions

At this point a complete restructuring of the docs ground up, which I know isn't what you're suggesting, wont be entertained until we hit a major release. Aside from that I love to discuss more about audience and how to tailor this content to be extremely beneficial to learners.

I love that phrase "zero to hero".
